.github/workflows/main.yaml

```diff
@@ -0,0 +1,16 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install -r requirements.txt
+      - run: mkdocs gh-deploy
```

Comment:
This looks like it handles publishing for merged updates. Ideally we would also do `mkdocs build` for PRs to help find issues before merging.